File: problem/Kotlin/richest-customer-wealth.kt
class Solution {
    fun maximumWealth(accounts: Array<IntArray>): Int {
        return accounts.map { it.sum() }.max() ?:0
    }
}

/*
class Solution {
    fun maximumWealth(accounts: Array<IntArray>) = Int.run{
        var big = 0;
        accounts.forEach{it ->
            var sum = 0;
            big = Math.max(big, it.sum())
        }
        big
    }
}
 */

File: problem/C++/richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int big = 0;
        int tmp = 0;
        for(auto i : accounts){
            tmp = 0;
            for(auto j : i)
                tmp += j;
            big = (big < tmp)? tmp: big;
        }   
        return big;
    }
};